scripts/validate_demux_files.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python3
+
+import click
+import pandas as pd
+from pathlib import Path
+from subprocess import run, CalledProcessError
+
+from mmeds.util import validate_demultiplex
+
+CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
+
+
+@click.command(context_settings=CONTEXT_SETTINGS)
+@click.version_option(version='0.1')
+@click.option('-d', '--data_dir',
+              help="path to reverse reads multiplexed file")
+@click.option('-m', '--map_path',
+              required=True,
```

Comment:
is this really the only one that's required? it seems like all of them are, currently

Reply:
yeah i should fix, thanks